Menus/MainMenu/MainMenu.cs

```diff
@@ -43,6 +44,8 @@ sealed class MainMenu : MonoBehaviour, IMainMenu, IConnectInterfaces, IInstantia
         MainMenuUI m_MainMenuUI;
         float m_LastRotationInput;
         MenuHideFlags m_MenuHideFlags = MenuHideFlags.Hidden;
+        float m_RotationInputStartValue;
+        DateTime m_RotationInputStartTime;
```

Comment:
Pretty unusual for us to use DateTime, normally we just use float since that is what Unity's Time class gives us